components/pipeline-service/production/base/pac-app-name.yaml

```
@@ -0,0 +1,4 @@
+---
+- op: replace
+  path: /data/application-name
+  value: Red Hat Trusted Application Pipeline
```

I think we should use the same name as the GitHub app. For projects with large contributor bases, many of the folks who submit PRs will only interact with RHTAP through the GitHub app/integrations:

```suggestion
  value: Red Hat Trusted App Pipeline
```

---

Done